frontend/src/components/Forms/MineRecordForm.js

```diff
@@ -104,6 +115,12 @@ export class MineRecordForm extends Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
```

Comment:
If you want to set the state because of changing props, use `componentWillReceiveProps` instead.

checking `this.props` against `nextProps`
There is already a pattern in place in this component that you can follow, inside the if/else you can call directly with the correct input`this.props.change(FORM.MINE_RECORD, "status_date", null)`

Reply:
Done